commitizen/cli.py

```diff
@@ -190,6 +190,20 @@
                         "default": None,
                         "help": "keep major version at zero, even for breaking changes",
                     },
+                    {
```

Comment:
Would it be better if we could extract these command value as another dict?

Reply:
Done 👍🏼